src/components/RoomSelection/RoomSelectionCards.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from "react";
+import {  Box, Typography, Card } from "@mui/material";
+import COLORS from "../../constants/CustomColors";
+import { iRoomCards } from "../../interfaces/interfaces";
+import { useNavigate } from "react-router-dom";
+export const RoomSelectionCards = ({
+    avaibility,
```

Comment:
typo

Reply:
I dont understand what is the exact problem